content/en/providers/check-in/sp/_index.md

```diff
@@ -531,15 +532,16 @@ UserInfo Endpoint:
-| ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `openid`                       | `sub`                                                                                                                                                                                             |
-| `profile`                      | <ul><li>`name`</li><li>`given_name`</li><li>`family_name`</li><li>`preferred_username`</li></ul>                                                                                                  |
-| `email`                        | <ul><li>`email`</li><li>`email_verified`</li><li>`voperson_verified_email`</li></ul>                                                                                                              |
-| `aarc`                         | <ul><li>`name`</li><li>`given_name`</li><li>`family_name`</li><li>`preferred_username`</li><li>`email`</li><li>`email_verified`</li><li>`voperson_verified_email`</li><li>`voperson_id`</li></ul> |
-| `eduperson_entitlement`        | `eduperson_entitlement`                                                                                                                                                                           |
-| `eduperson_scoped_affiliation` | `eduperson_scoped_affiliation`                                                                                                                                                                    |
-| `voperson_id`                  | `voperson_id`                                                                                                                                                                                     |
+| Scope                           | Claims                                                                                                                                                                                                                                                                                                               |
+| ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `openid`                        | <ul><li>`sub`</li><li>`eduperson_assurance`</li></ul>                                                                                                                                                                                                                                                                |
```

Comment:
I suggest we remove eduperson_assurance from here (it's already mentioned in the user attributes/claims section

Reply:
Done